src/sftp.rs

```diff
@@ -432,7 +432,9 @@ impl Drop for Sftp {
             let sess = inner.sess.lock();
             let was_blocking = sess.is_blocking();
             sess.set_blocking(true);
-            assert_eq!(unsafe { raw::libssh2_sftp_shutdown(inner.raw) }, 0);
+            // The shutdown statement can go wrong and return an error code, but we are too late 
```

Comment:
It would be nice to log the error, but ssh2-rs does not configure a logger. I can volunteer setting up an `envlogger` and add a bit of debug statements around if @wez agrees.

Reply:
I'm not sure that it's worth it; I suspect that this event is really an issue in the underlying library and that there isn't anything that we can in this crate or the downstream consumers, and that logging it will end up being a fairly annoying "so what?" kind of thing!

Reply:
Yep, I see your point.. Thanks for merging!